src/main/webapp/WEB-INF/tags/page.tag

```diff
@@ -29,7 +29,7 @@
     <script type="text/javascript" src="${data.jQueryUiFilePath}"></script>
     <script type="text/javascript" src="/js/common.js"></script>
     <script type="text/javascript" src="/bootstrap/js/bootstrap.min.js"></script>
-
+    <script type="text/javascript" src="/js/jquery.highlight.js"></script>
```

Comment:
This library is used only by admin search page, right? If yes it shouldn't be in the main `page.tag` (which is used everywhere).